vita3k/app/src/app_init.cpp

```diff
@@ -118,44 +118,60 @@ void init_paths(Root &root_paths) {
 #if defined(__linux__) && !defined(__ANDROID__) && !defined(__APPLE__)
     // XDG Data Dirs.
     auto env_home = getenv("HOME");
-    auto env_data_dirs = getenv("XDG_DATA_DIRS");
-    auto env_data_home = getenv("XDG_DATA_HOME");
-    auto env_cache_home = getenv("XDG_CACHE_HOME");
-    auto env_config_home = getenv("XDG_CONFIG_HOME");
+    auto XDG_DATA_DIRS = getenv("XDG_DATA_DIRS");
+    auto XDG_DATA_HOME = getenv("XDG_DATA_HOME");
+    auto XDG_CACHE_HOME = getenv("XDG_CACHE_HOME");
+    auto XDG_CONFIG_HOME = getenv("XDG_CONFIG_HOME");
 
-    if (env_data_home != NULL)
-        root_paths.set_pref_path(fs::path(env_data_home) / app_name / app_name / dir_sep);
+    if (XDG_DATA_HOME != NULL)
+        root_paths.set_pref_path(fs::path(XDG_DATA_HOME) / app_name / app_name / dir_sep);
```

Comment:
pref_path needs to move to a subdir (I voted for /eMMC), or the fallback paths for static_assets_path and shared_path below are going to collide with it. (I.e. Non-vita files will end up in pref_path.)

Reply:
pref_path is already in a subdir (`.local/share/Vita3K/Vita3K`), i like the idea of changing it to eMMC, but i just want to know what the others think about it first

Reply:
Maybe that should be a different PR then?